src/Containers/FeaturedProfilesCard.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import styled from 'styled-components';
+import { AddUser } from '@styled-icons/entypo/AddUser';
+import { flex } from '../Utils/Mixins';
+
+export interface IProfile {
+    image: string;
+    initials: string;
+    id: number;
+}
+
+export interface IFeaturedProfilesCardProps {
+    profileData: IProfile[];
+    alt: string;
+    width?: number;
+    height?: number;
+}
+
+export const FeaturedProfilesCard: React.FC<IFeaturedProfilesCardProps> = ({
+    alt,
+    profileData,
+    width = 100,
+    height = 100,
+}): React.ReactElement => {
+    const getProfileCircles = (): any => {
+        const PROFILE_PICTURE_LIMIT = 3;
+        const PROFILE_INITIALS_START_INDEX = 2;
+        const PROFILE_INITIALS_END_INDEX = 5;
+        const REMAINING_PROFILES_INDEX = 5;
+        const REMAINING_PROFILES_VALUE = profileData.length - 5;
+        const MAX_PROFILES = 7;
+        const START_FROM_PROFILE_INDEX = 0;
+
+        const UserProfilePicture = (profile: any): React.ReactElement => {
+            return (
+                <CircleImage background="none" key={profile.image}>
+                    <img
+                        src={profile.image}
+                        alt={alt}
+                        height={height}
+                        width={width}
+                    />
+                </CircleImage>
+            );
+        };
+
+        const UserInitials = (profile: any): React.ReactElement => {
+            return (
+                <CircleImage background="orange">
+                    <CircleContent>{profile.initials}</CircleContent>
+                </CircleImage>
+            );
+        };
+
+        const remainingUserProfiles = (): React.ReactElement => {
+            return (
+                <CircleImage background="grey">
+                    <CircleContent>{REMAINING_PROFILES_VALUE}</CircleContent>
+                </CircleImage>
+            );
+        };
+
+        const addUserIcon = (): React.ReactElement => {
+            return (
+                <CircleImage background="grey">
+                    <Icon as={AddUser} />
+                </CircleImage>
+            );
+        };
+
+        return profileData
+            .map(
+                (profile, index): React.ReactElement => {
+                    if (index < PROFILE_PICTURE_LIMIT) {
+                        return UserProfilePicture(profile);
+                    }
+                    if (
+                        index > PROFILE_INITIALS_START_INDEX &&
+                        index < PROFILE_INITIALS_END_INDEX
+                    ) {
+                        return UserInitials(profile);
```

Comment:
why is the function in caps?
Please look at the JavaScript Style Guide on Confluence